scripts/sentinel.sh

```diff
@@ -252,3 +255,29 @@ for gibs_id_dir in "$gibs_dir"/* ; do
     fi
 done
 echo "All GIBS tiles created"
+
+# Generate VI files
+echo "Generating VI files"
+vi_generate_indices -i "$workingdir" -o "$vidir" -s "$outputname"
+vi_generate_metadata -i "$workingdir" -o "$vidir"
+
```

Comment:
@sharkinsspatial I'm seeing `--include '*_stac.json'` in the S3 copy on the lines below, but I haven't seen STAC produced in the test outputs we've been generating.

Assuming we want to do that at this point, I think we need to add something like,

```suggestion
vi_generate_metadata -i "$workingdir" -o "$vidir"
vi_generate_stac_items --cmr_xml "$vidir/${vi_outputname}.cmr.xml" --endpoint data.lpdaac.earthdatacloud.nasa.gov --version 020 --out_json "$vidir/${vi_outputname}_stac.json"

```
to generate the `*_stac.json`. If this sounds right would you mind double checking I'm getting the filename arguments correct? I'm not totally sure I got those correct in the comment suggestion and wouldn't mind a double check before comitting